dynamiqs/integrators/core/diffrax_integrator.py

```diff
@@ -201,11 +201,11 @@ def terms(self) -> dx.AbstractTerm:
 
         def vector_field(t, y, _):  # noqa: ANN001, ANN202
             tmp = -1j * self.H(t) @ y
-            for L in self.Ls:
-                Lt = L(t)
-                Ltdag = Lt.dag()
-                Lt_y = Lt @ y
-                tmp += 0.5 * (Lt_y @ Ltdag - Ltdag @ Lt_y)
+            Ls = [L(t) for L in self.Ls]
+            for L in Ls:
+                Ldag = L.dag()
+                L_y = L @ y
+                tmp += 0.5 * (L_y @ Ldag - Ldag @ L_y)
```

Comment:
I don't think this commit is useful. You're storing all L(t) values instead of storing them one by one. The code is more elegant, but I don't think it's faster and it's less memory-efficient.

But in any case, I'll try to rework this implementation to find something faster.

Reply:
Yes, it was only for style 😇